Issue-Label-Bot-v2/tests/test_mlp.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import sys
+sys.path.insert(1, '../notebooks')
```

Comment:
If you want you can add a symlink and it will work on GitHub as well that way you don't have to worry about changing your python path.  Not necessary just a suggestion to make your life easier

Reply:
That sounds good. I will have a look.